views/views_aggregator_plugin_style_table.inc

```diff
@@ -728,6 +735,64 @@ class views_aggregator_plugin_style_table extends views_plugin_style_table {
     return is_array($rendered_value) ? backdrop_render($rendered_value) : $rendered_value;
   }
 
+  /**
+   * Format a raw timestamp according to the supplied date handler settings.
+   *
+   * @param object $field_handler
+   * @param int $value
+   *
+   * @return string, date formatted according to Views handler settings
+   *
+   * Note: this was taken in part from views_handler_field_date::render($values)
+   */
+  function render_date($field_handler, $value) {
+    $format = $field_handler->options['date_format'];
```

Comment:
The following code was added to Backdrop's views_handler_field_date::render(), which is where the code for the method here came from. This should be added to your PR, please.

```suggestion
    if (!is_numeric($value)) {
      // If the value isn't numeric, assume it's an SQL DATETIME.
      $value = strtotime($value);
    }
    $format = $field_handler->options['date_format'];
```